.github/workflows/release.yaml

```diff
@@ -15,11 +15,11 @@ jobs:
         # docs: https://github.com/marketplace/actions/golang-cgo-cross-compiler#inputs
         with:
           xgo_version: latest
-          go_version: 1.17
+          go_version: 1.18
```

Comment:
We don't nor want to use any new language features that come with golang 1.18 yet and I don't think this version includes something like security patches, so why is this version upgrade required? The parts of the code base that are written in go were also written targeting go 1.17 at the time. *If* we decide to upgrade to 1.18 here, we should also update `go.mod` and make sure the code still compiles in development.

Reply:
`go.mod` just list the lowest required version, so we do NOT need to change it.

1.18 runtime has hands of optimization and compatibility enhancement for multiple architecture, which is really useful when it comes to ARM.

Reply:
Oh I didn't realize 1.18 brings hands of optimization and compatibility enhancements. Alright, then!